frame/assets/src/lib.rs

```diff
@@ -481,7 +485,7 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			#[pallet::compact] id: T::AssetId,
 			witness: DestroyWitness,
```

Comment:
@shawntabrizi whats the point of having a witness?

Reply:
Witness data is used to estimate the weight needed to execute that extrinsic without needing to read storage to actually know how many elements there are.